policyengine_canada/variables/gov/provinces/sk/tax/income/credits/pension_income_amount/sk_pension_income_credit.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_canada.model_api import *
+
+
+class sk_pension_income_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Saskatchewan pension income credit"
+    unit = CAD
+    definition_period = YEAR
+    reference = (
+        "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-22e.pdf#page=1",
+        "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-23e.pdf#page=1",
+        "https://publications.saskatchewan.ca/api/v1/products/583/formats/806/download#page=16",
+    )
+    defined_for = ProvinceCode.SK
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.sk.tax.income.credits.pension_income_amount
+        # Payments exclude Canada Pension Plan, Quebec Pension Plan, Old Age Security, or Guaranteed Income Supplement payments
```

Comment:
@MaxGhenis it seems to be more accurate to merge the GIS and OAS in (Alex has created the base PR a while ago, we would need to refine it). What do you think?

Reply:
yeah sounds good